livelossplot/outputs/matplotlib_plot.py

```diff
@@ -123,9 +123,11 @@ def _draw_metric_subplot(self, ax: plt.Axes, group_logs: Dict[str, List[LogItem]
         for name, logs in group_logs.items():
             if len(logs) > 0:
                 xs = [log.step for log in logs]
-                xs = xs[min(self.start_epoch, len(xs) - self.start_epoch):]
+                x_first = min(self.from_epoch, len(xs) - self.from_epoch)
+                xs = xs[x_first:]
                 ys = [log.value for log in logs]
-                ys = ys[min(self.start_epoch, len(ys) - self.start_epoch):]
+                y_first = min(self.from_epoch, len(ys) - self.from_epoch)
```

Comment:
Is it possible that `y_first != x_first`?

Reply:
no, done